### DISCUSSION TITLE
Multi container on AKS

### DISCUSSION THREAD

**Discussion by Foxrider721** [WeblateOrg/weblate#7532] [General]
Hello everyone ! 

I currently trying to install weblate on kubernetes, I want to be able
to scale weblate horizontally in case of higher pressure, I am using
the charts you made and when i choose for the persistence
READWRITEMANY, to be able to share the /app/data directory among
weblate container (as it is written in your documentation with docker
container), but the file beat-schedule is deleted.
I am using AKS and azure file system standard LRS as a volume

I found a few documentation on this problem, celery appears not to
support certain file system especially network shared ones

Otherwise when I use the READWRITEONCE with a volume created with disk
and it's running perfectly,

Can you help me with this issue please ?

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/admin/install/docker.html#scaling-horizontally
